packages/x-tailwindcss/src/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './helpers';
-export { default } from './plugin';
+export { default as xTailwindPlugin } from './plugin';
```

Comment:
This si a Breaking Change. Should it be added as such in the merge? Are we doing that for the `x-tailwindcss`?